Makefile

```diff
@@ -35,3 +35,16 @@ checkstyle: black lint
 
 .PHONY: test
 test: only-test checkstyle
+
+venv = qem-bot
+environment:
+	if [ ! -d "$(venv)" ]; then virtualenv -p python3 $(venv); fi
```

Comment:
Where should the application "venv" come from? `$(venv)` would be the invocation of an application "venv" that should return a directory path. I don't see where this would come from